letsencrypt/cli.py

```diff
@@ -1051,6 +1030,51 @@ def _plugins_parsing(helpful, plugins):
 
     helpful.add_plugin_args(plugins)
 
+    # These would normally be a flag within the webroot plugin, but because
+    # they are parsed in conjunction with --domains, they live here for
+    # legibiility. helpful.add_plugin_ags must be called first to add the
+    # "webroot" topic
+    helpful.add("webroot", "-w", "--webroot-path", action=WebrootPathProcessor,
```

Comment:
I disagree with moving these flags outside of `webroot.py`. We have a very clear infrastructure for adding plugin specific flags. The plugin defines `add_parser_arguments` and all arguments needed by the plugin are defined there.

This completely disregards that infrastructure and creates an unnecessary exception to the pattern for `webroot`.